src/main/kotlin/app/extractors/CExtractor.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Liubov Yaronskaya (lyaronskaya@sourcerer.io)
+
+package app.extractors
+
+import app.model.CommitStats
+import app.model.DiffFile
+
+class CExtractor : ExtractorInterface {
+    override fun extract(files: List<DiffFile>): List<CommitStats> {
+        TODO("not implemented")
+    }
+
+    override fun extractImports(fileContent: List<String>): List<String> {
+        val libraries = mutableSetOf<String>()
+
+        val quoteRegex = Regex("#import\\s+\"(\\w+)[/\\w+]*\\.\\w+\"")
```

Comment:
Did you test it? Probably shouldn't work.
Also, you can use only one regex with OR: `#import\s+[\"|<](\w+)[/\w+]*.\w+[\"|>]`

Reply:
Fix double slash everywhere.

Reply:
`#include` maybe?

Reply:
Fixed.